packages/create-starknet/templates/next/pages/index.tsx

```diff
@@ -0,0 +1,116 @@
+import Head from 'next/head'
```

Comment:
Can we customize this  page so that it does something starknet-specific? Like show the current block number and a button to connect wallet (or if connected, with the connected address).

Reply:
Sure, I'm on it !